### DISCUSSION TITLE
Broken image in Watchface broke something bad: GTS-4 stuck at coupling screen

### DISCUSSION THREAD

**Discussion by ojde** [zepp-health/zeppos-forum#303] [Q&A]
While developing a watchface (coding, not website), I added an image widget with a faulty .tga (RLEncoded instead of uncompressed), which sent the GTS-4 into a boot loop after loading the watchface.zip onto it (it was the only change to a previously working watchface, so I'm sure it was that).

The boot loop was, after quite a while, interupted with a prompt to restore system (hold button pressed three times). After that I could successfully recouple the watch with the zepp app, but the watch itself is now stuck at the coupling screen (see image below) and does not react to swipes or button presses - but turns off screen after 10 sec and turns it on on button press or wrist movements, also vibrates on incoming messages - but never shows anything but the coubling screen. Zepp app functions seem to work, too (changing settings, sync, etc).

Trying to change the watchface in the Zepp app leads to an error saying that the watchface memory is full, to please delete first.

What can I do to get the watch back to work?

![threema-20240209-231425946](https://github.com/zepp-health/zeppos-forum/assets/1812959/d9a33d9a-c842-4f81-a9a2-a81e645946f6)

**Comment by MyoungXUE** [comment]
Hello @ojde, sorry I didn't notice this discussion before

1. Please connect the power supply to ensure that the device is charged
2. Press and hold the Home button, the charging animation disappears after about 10-20 seconds, release the Home button after the charging animation disappears, and press the Home button again for about two seconds, and a prompt to reset the system will pop up
3. Follow the prompts to transfer the installation package to the watch through Zepp and reinstall the system

**Comment by ojde** [comment]
Thanks for the response. As it happens, the watch fixed itself after a while: The Zepp app announced an update, installed it and the watch booted all right. So I'm unable to confirm that your instructions would have helped me, too. But I'll keep the  in mind if I - heavens forbid - manage to get myself in trouble again.
